projects/ssr-tests/src/utils/proxy.utils.ts

```diff
@@ -6,29 +6,9 @@
  */
 
 import * as http from 'http';
-import httpProxy, { ProxyResCallback } from 'http-proxy';
+import httpProxy from 'http-proxy';
 
-const proxy = httpProxy.createProxyServer({
-  secure: false,
-  // selfHandleResponse: true,
-});
-
-/**
- * Default settings to send http requests.
- */
-const REQUEST_OPTIONS = {
-  host: 'localhost',
-  port: 4000,
-};
-
-/**
- * Response from SSR server.
- */
-export interface SsrResponse {
-  statusCode: number | undefined;
-  headers: http.IncomingHttpHeaders;
-  body: string;
-}
+let proxy: httpProxy;
```

Comment:
Can't we define const in line 37 - locally in the scope of the function `startBackendProxyServer` ?